src/helpers/startCountdown.test.ts

```diff
@@ -93,7 +93,7 @@ describe("startCountdown", () => {
 					isTimerPaused: false,
 				});
 
-				expect(clearIntervalSpy).toHaveBeenCalledTimes(2);
+				expect(clearIntervalSpy).toHaveBeenCalledTimes(1);
```

Comment:
This line (96) is correct.

However, since you moved the clearInterval to inside the `clientTimerStore.timer = setInterval()`, we need to add an additional test for it. 

Basically, when the timer is already running, when it gets to 0, the clearInterval should be called (a total of 2 times; 1 for when `startCountdown` is called, the other for when `durationInSeconds` is ticking down to 0 inside the `clientTimerStore.timer = setInterval()`.

You can add this piece of code to line 127. This will be inside the describe `when the durationInSeconds is greater than 0`

![image](https://github.com/CommunityFocus/cf-frontend/assets/45009203/c1a458b5-9604-4d82-b137-ca138af229f0)


```
			describe("when the timer ticks to 0", () => {
				it("should clear the timer", () => {
					startCountdown({
						durationInSeconds,
						clientTimerStore,
						setTimestamp: mockSetTimestamp,
						isTimerPaused: false,
					});
					jest.advanceTimersByTime(10000);

					expect(clearIntervalSpy).toHaveBeenCalledTimes(2);
				});
			});


```